### PR TITLE
[RAPPS] Fix attempt to delete an already destroyed ImageList

### PR DESCRIPTION
**Pull request by KyleKatarn31**
## Purpose

- When closing RAPPS, attempt is made to destroy ImageList that have already been destroyed. This is due to explicit call ImageList_Destroy while these ImageList are also linked to a CMainToolbar object which also tries to delete these ImageList in TOOLBAR_Destroy (inherited from comctl32 toolbar)

JIRA issue: [CORE-17295](https://jira.reactos.org/browse/CORE-17295)

## Proposed changes

- Remove unneeded calls to ImageList_Destroy

### CODE REVIEW

File: base/applications/rapps/appview.cpp [hunk 1]
```diff
@@ -141,7 +141,7 @@ HWND CMainToolbar::Create(HWND hwndParent)
         return FALSE;
     }
 
-    ImageList_Destroy(SetImageList(hImageList));
+    SetImageList(hImageList);
```
Comment (learn-more) [inline base/applications/rapps/appview.cpp:144]: This destroys the previous imagelist.

Comment (KyleKatarn31) [reply base/applications/rapps/appview.cpp:144]: Yes but TOOLBAR_Destroy will later try to destroy the same Imagelist, as set as component Internal Imagelist by SetImageList (inherited from CToolbar).
Same problem as for https://github.com/reactos/reactos/pull/3211 (where the issue came from the use of a ListView)

Comment (learn-more) [reply base/applications/rapps/appview.cpp:144]: Please explain how TOOLBAR_Destroy will delete the imagelist that is being leaked here.
Does it keep a list of previously returned imagelists?

Comment (KyleKatarn31) [reply base/applications/rapps/appview.cpp:144]: - hImageList get referenced as himlInt in the Toolbar component when SetImageList is called
- then destroyed L144 by ImageList_Destroy
- and himlInt gets deleted again when TOOLBAR_Destroy is called (when component is destroyed at RAPPS closure)

Step 2 is a bug.

**Comment by KyleKatarn31** [comment]
Close as I can't reproduce with 0.4.15-dev-989-gb4e07e

**Pull request closed without merging**